### PR TITLE
Add notes for 1.x and 5.x Chocolatey components

### PR DESCRIPTION
**Pull request by gep13**
## Description Of Changes

Add release notes for releases of all Chocolatey components

## Motivation and Context

There are releases coming up, and we need to get this documented

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

N/A

### CODE REVIEW

File: input/en-us/choco/release-notes.md [hunk 1]
```diff
@@ -19,6 +19,17 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
 <?! Include "../../shared/chocolatey-component-dependencies-link.txt" /?>
 
 
+## 1.4.0 (May 10, 2023)
+
+### Bug Fix
+
+- Fix - When no hook scripts and installed you cannot install/upgrade a package on a system using PowerShell 2 - see [#3088](https://github.com/chocolatey/choco/issues/3088).
```
Comment (JPRuskin) [inline input/en-us/choco/release-notes.md:26]: I feel like this should be "are installed", and I feel a sense of deja vu saying that...

File: input/en-us/choco/release-notes.md [hunk 2]
```diff
@@ -19,6 +19,17 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
...
+
+### Improvement
+
+- Ensure the most recent version of Chocolatey-NuGet.Core package is being used - see [#3150](https://github.com/chocolatey/choco/issues/3150).
```
Comment (AdmiringWorm) [inline input/en-us/choco/release-notes.md:30]: Do we want to mention why it is needed here in the release notes?

File: input/en-us/choco/release-notes.md (hunk 1 above)
Comment (gep13) [reply input/en-us/choco/release-notes.md]: Thank you for catching that!

File: input/en-us/choco/release-notes.md (hunk 2 above)
Comment (gep13) [reply input/en-us/choco/release-notes.md:30]: I am inclined to say no.  In the linked issue, there is another link through to the NuGet.Core issue, where the information is kept.  Adding more information here moves us away from the automation that we are trying to put in play for these release notes.

Comment (AdmiringWorm) [reply input/en-us/choco/release-notes.md:30]: fair enough

**Review by AdmiringWorm — CHANGES_REQUESTED** [review]
Can you rebase against latest master, the PR shows up as out of date

**Review by AdmiringWorm — APPROVED** [review]
LGTM

**Comment by gep13** [comment]
@AdmiringWorm @JPRuskin thank you for the reviews.  I have converted this back to draft for now, and will merge once all the packages have been released.

**Review by JPRuskin — DISMISSED** [review]

File: input/en-us/agent/release-notes.md [hunk 3]
```diff
@@ -31,6 +31,13 @@ This covers the release notes for the Chocolatey Agent Service (`chocolatey-agen
 <?! Include "../../shared/chocolatey-component-dependencies.txt" /?>
 
 
+## 1.1.2 (May 10, 2023)
+
+### Bug
```
Comment (pauby) [inline input/en-us/agent/release-notes.md:36]:
```suggestion
### Bug Fix
```

File: input/en-us/chocolatey-gui/release-notes.md [hunk 4]
```diff
@@ -19,6 +19,13 @@ This covers changes for the "chocolateygui" package, which is available as FOSS.
 <?! Include "../../shared/chocolatey-component-dependencies.txt" /?>
 
 
+## 1.1.3 (May 10, 2023)
+
+### Bug
```
Comment (pauby) [inline input/en-us/chocolatey-gui/release-notes.md:24]:
```suggestion
### Bug Fix
```

File: input/en-us/licensed-extension/release-notes.md [hunk 5]
```diff
@@ -43,6 +43,13 @@ Please see [Install the Licensed Edition](xref:setup-licensed) for information o
 <?! Include "../../shared/chocolatey-component-dependencies-link.txt" /?>
 
 
+## 5.0.3 (May 10, 2023)
+
+### Bug
```
Comment (pauby) [inline input/en-us/licensed-extension/release-notes.md:48]:
```suggestion
### Bug Fix
```

**Review by pauby — CHANGES_REQUESTED** [review]

**Pull request merged**